src/components/organisms/modals/ConfirmationModal/ConfirmationModal.tsx

```diff
@@ -18,6 +18,7 @@ const ConfirmationModal: FC<ConfirmationModalBaseProps> = ({
   return (
     <ConfirmationModalBase
       {...rest}
+      className={className}
```

Comment:
**[non-blocker]**

className is not directly used inside this component, so would be better to pass it inside ...rest with all of the other props that are just passed through